### PR TITLE
KeywordBearTest: Test for KeywordBear offering all patches in results

### PR DESCRIPTION
**Pull request by namanyadav12**
Add test for https://github.com/coala/coala-bears/issues/1274
The test checks the diff to make sure that it proposes a
patch to remove the comment even for multiple comments.

Closes https://github.com/coala/coala-bears/issues/1306

### CODE REVIEW

File: tests/general/KeywordBearTest.py [hunk 1]
```diff
@@ -84,6 +84,31 @@ def test_keyword_not_in_comment(self):
...
+    def test_keyword_in_multiple_comments(self):
+        text = ['# todo 1\n', '# todo 2']
+        comments = [SourceRange.from_values('F', 1, 1, 2, 8)]
+        dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type({'comments': comments})
+            ]
+        }
+
+        with execute_bear(self.uut, 'F', text,
```
Comment (Techievena) [inline tests/general/KeywordBearTest.py]: Please use named keywords here. Refer line 224 of this file. Also please edit the lines in other test cases where named keywords are not used.

File: tests/general/KeywordBearTest.py [hunk 2]
```diff
@@ -84,6 +84,31 @@ def test_keyword_not_in_comment(self):
                           dependency_results=dep_results) as result:
             self.assertEqual(len(result[0].diffs), 0)
 
+    def test_keyword_in_multiple_comments(self):
```
Comment (Techievena) [inline tests/general/KeywordBearTest.py]: better if `test_diff_keyword_in_multiple_comments`

File: tests/general/KeywordBearTest.py (hunk 1 above)
Comment (Techievena) [inline tests/general/KeywordBearTest.py]: Ya better file a newcomer issue for this :+1:

**Comment by namanyadav12** [comment]
@Techievena i think you need to ack the commit in a comment and not in a review for it to pass on review/gitmate/manual :)

**Comment by Techievena** [comment]
Not got that right yet. :worried: Only if maintainers approve changes then only it passes.

**Comment by namanyadav12** [comment]
That's not what I meant. If you write ack commit in a review like you did above, it will have no effect . You need write it in a comment, then only it will pass all 7 checks and then a maintainer can merge.

**Comment by Techievena** [comment]
ack 5c8620f

**Comment by Techievena** [comment]
unack 5c8620f

**Comment by Techievena** [comment]
Have you rebased it.Its been a long time since last PR

**Comment by rhemon** [comment]
ack 34a68b4

**Comment by rhemon** [comment]
sorry, looks like you need to rebase again @namanyadav12 
unack 34a68b4

**Comment by rhemon** [comment]
unack 34a68b4

**Comment by supergr35** [comment]
Hello, can you please rebase this PR if you're still working on it :)

**Pull request closed without merging**